src/Illuminate/Foundation/Console/Tinker/Presenters/IlluminateApplicationPresenter.php

```diff
@@ -0,0 +1,72 @@
+<?php namespace Illuminate\Foundation\Console\Tinker\Presenters;
+
+use Psy\Presenter\ObjectPresenter;
+use Illuminate\Foundation\Application;
+
+class IlluminateApplicationPresenter extends ObjectPresenter {
+
+	/**
+	 * Illuminate Application methods to include in the presenter.
+	 *
+	 * @var array
+	 */
+	protected static $appProperties = [
```

Comment:
That's a good question that I don't know the answer for. Maybe have Application implement `__debugInfo` then just blacklist the dumb keys in that, rather than whitelisting methods here?